src/plays/youtube-clone/Readme.md

```diff
@@ -0,0 +1 @@
+# Youtube-Clone
```

Comment:
Please provide details in your Readme file. You can refer to the readme file from other plays as a reference.